src/towncrier/test/test_build.py

```diff
@@ -1631,6 +1631,29 @@ def test_invalid_fragment_name(self, runner):
         self.assertEqual(1, result.exit_code, result.output)
         self.assertIn("Invalid news fragment name: feature.124", result.output)
 
+    @with_project(
+        config="""
+        [tool.towncrier]
+        package = "foo"
+        template = ["template.jinja"]
+        """
+    )
+    def test_ignored_template_string(self, runner):
+        """
+        Files used in `template` are automatically ignored.
```

Comment:
Can there be more template files ?

I am reading the docs from [here](https://towncrier.readthedocs.io/en/latest/configuration.html) 

```
template

Path to the template for generating the news file.
If the path looks like <some.package>:<filename.ext>, it is interpreted as a template bundled with an installed Python package.
"towncrier:default.rst" by default unless filename ends with .md, in which case the default is "towncrier:default.md".
```

Reply:
I think this is an irrelevant case, because to my understanding it'd pull the said file from a Python package, meaning it's not in your current directory/where changes lie.

Reply:
Why the test was passing with `template = ["template.jinja"]`

Is the test valid ?

Was the test failing before applying the patch in `_builder.py` ?

Reply:
Yes, it was failing.

Reply:
Perfect. I was a bit confuse of why we have the initial version of the test.

I double check and it looks good.